rpc_jobs/periodic_cleanup.yml

```diff
@@ -25,29 +25,53 @@
       - build-discarder:
           days-to-keep: 3
     dsl: |
-      node(){
-        deleteDir()
-        dir("rpc-gating") {
-          stage("Prepare"){
-            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
-            common = load 'pipeline_steps/common.groovy'
-          }
-          stage("Cleanup Jenkins Workspaces"){
-            sh "scripts/workspace_cleanup.sh"
-          }
-          stage("Cleanup Jenkins TMP Dir"){
-            sh "scripts/tmp_cleanup.sh"
-          }
-          stage("Cleanup Docker Objects"){
-            sh "scripts/docker_cleanup.sh"
+      // Get list of jenkins slaves
+      @NonCPS
+      def getNodes() {
+        return jenkins.model.Jenkins.instance.nodes.grep {
+          node -> node.name =~ /^(long-|master|rpc-jenkins-n)/
+        }.collect {
+          node -> node.name
+        }
+      }
+
+      // run node cleanups on all nodes
+      def nodes = getNodes()
+      def branches = [:]
```

Comment:
Use of the term `branches` is probably not ideal given that generally in the repo it refers to Git branches.